### PR TITLE
install gems only for the user, and set a vendor path for bundle use

### PR DESCRIPTION
**Pull request by tompscanlan**
Using a mac, I needed to do this to test a change in the docs.

* `gem ... --user-install` to avoid using sudo
* use a vendor dir that is outside of the docs tree

Signed-off-by: Tom Scanlan <tscanlan@vmware.com>

### CODE REVIEW

**Comment by tompscanlan** [comment]
@pksrc that would be fine.  let me update and see if it makes more sense.

File: docs/README.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ rbenv install 2.6.3
 gem install bundler
 ```
 
+*Note: if you hit a permissions error for the `gem install bundler` follow advice from the [bundler docs](https://bundler.io/doc/troubleshooting.html#permission-denied-when-installing-bundler)*
```
Comment (pksrc) [inline docs/README.md:65]: This looks good. However, I noticed this link does not talk about the `vendor/cache/` which you had mentioned w/ the first change. Let me know if that was required for you to build the website along with the user install?

Comment (tompscanlan) [reply docs/README.md:65]: Turns out that I didn't need to change the vendor cache location.  default seems to be fine.

**Comment by pksrc** [comment]
@tompscanlan My apologies but I forgot to have you squash the commits before merging. Since the changes in the PR are related and can be referenced with a single commit, I need to revert the merge. Please ACK if you can resubmit the PR with the commits squashed.

**Pull request merged**